Streetcode/Streetcode.BLL/DTO/Authentication/Login/LoginRequestDTO.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using Streetcode.BLL.Attributes.Authentication;
+using Streetcode.BLL.Constants.Authentication;
+
+namespace Streetcode.BLL.DTO.Authentication.Login;
+
+public class LoginRequestDTO
+{
+    [Required]
+    [ValidEmail]
+    [EmailAddress]
```

Comment:
Here a added **[ValidEmail]** in case we have some specific email validation( that goes out of bounds of format checking). Now **[ValidEmail]** has no specific logic of email chacking that is different from **[EmailAddress]**, but things may change in future